### DISCUSSION TITLE
GPU for Apple Silicon

### DISCUSSION THREAD

**Discussion by fbarfi** [aesara-devs/aesara#857] [Q&A]
Just wondering whether there might be a possibility to use the Apple M1 GPU cores in aesara (now or in the near future) ? I have had so far good success with the CPU cores of  MacBookPro M1 Max. Thanks.

**Comment by ricardoV94** [comment]
That would depend on whether JAX supports Apple GPU, since currently Aesara only provides GPU functionality via the JAX backend
